src/Microsoft.Fx.Portability/Microsoft.Fx.Portability.csproj

```diff
@@ -14,25 +14,22 @@
     <PackageReference Include="Newtonsoft.Json" Version="9.0.1" />
     <PackageReference Include="Microsoft.AspNetCore.WebUtilities" Version="1.1.1" />
     <PackageReference Include="System.Net.Http" Version="4.3.2" />
+    <PackageReference Include="System.Collections.Immutable" Version="1.3.1" />
```

Comment:
I remember @alinapopa  moving this into the netstandard1.3 and net46 TargetFrameworks because at runtime, it would fail to resolve the System.Collections.Immutable dependency.  Does it still work in both VS2017 and VS2015?

Reply:
Yup - the issue was that we hadn't been including System.Collections.Immutable with the VSIX. It's strongly named and we're not sharing it with other systems (such as Roslyn) so we should be okay. It works in all cases

Reply:
In this case is the net46 target still needed?

Reply:
probably not - I won't remove that target right now so this change won't be blocked, but we can look into it.